go.mod

```diff
@@ -3,6 +3,9 @@ module github.com/moby/buildkit
 go 1.22.0
 
 require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.17.0
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.8.1
+	github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.6.0
```

Comment:
@tonistiigi I think we should align go-actions-cache to v1.6.0 as well: https://github.com/tonistiigi/go-actions-cache/blob/bd99cf5bbc6567734a75c09ae87883d304eea6d1/go.mod#L6

Reply:
@vangarp Looking at peer deps from https://github.com/tonistiigi/go-actions-cache/pull/30 and to avoid binring unattended bumps, I think we should pin to `github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.5.0` and align other sdk deps: https://github.com/Azure/azure-sdk-for-go/blob/sdk/storage/azblob/v1.5.0/sdk/storage/azblob/go.mod#L6-L7

Reply:
do let me know. open to reverting or keeping this way (on 1.6.0 for azblob and 1.17.0 for azcore)

Reply:
> @vangarp Looking at peer deps from [tonistiigi/go-actions-cache#30](https://github.com/tonistiigi/go-actions-cache/pull/30) and to avoid bringing unattended bumps, I think we should pin to `github.com/Azure/azure-sdk-for-go/sdk/storage/azblob v1.5.0` and align other sdk deps: https://github.com/Azure/azure-sdk-for-go/blob/sdk/storage/azblob/v1.5.0/sdk/storage/azblob/go.mod#L6-L7

sounds good